File: webstore/webstore-java/src/main/java/com/renan/webstore/dto/UserDto.java
package com.renan.webstore.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class UserDto {
    private String email;
    private String jwt;
    private Integer id;
    private Integer status;
    private Integer expiresIn;
    private String name;
    private String lastName;
    private List<String> roles;
}

File: webstore/webstore-kotlin/src/main/kotlin/com/renan/webstore/dto/UserDTO.kt
package com.renan.webstore.dto

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonInclude.Include

@JsonInclude(Include.NON_NULL)
data class UserDTO(
    val email: String,
    val jwt: String,
    val id: Int,
    val active: Boolean,
    val expiresIn: Int,
    val name: String,
    val lastName: String?,
    val roles: List<String>?
)
